build.zig

```diff
@@ -8,6 +8,40 @@ const CrossTarget = std.zig.CrossTarget;
 const fs = std.fs;
 const Mode = builtin.Mode;
 
+/// The enumeration of tests with the unit test and all the runtime tests.
+const TestMode = enum {
+    /// This is the default test. This will run the unit tests.
+    UNIT,
+
+    /// This will run all the runtime tests below. The ALL_RUNTIME literal shouldn't be passed to
+    /// the OS code.
+    ALL_RUNTIME,
+
+    /// Run the OS's initialisation runtime tests to ensure the OS is properly set up.
+    INITIALISATION,
+
+    /// Run the panic runtime test.
+    PANIC,
+
+    ///
+    /// Return a string description for the test mode provided.
+    ///
+    /// Argument:
+    ///     IN mode: TestMode - The test mode.
+    ///
+    /// Return: []const u8
+    ///     The string description for the test mode.
+    ///
+    pub fn getDescription(mode: TestMode) []const u8 {
+        return switch (mode) {
+            .UNIT => "Run the unit tests",
+            .ALL_RUNTIME => "Run all the runtime tests",
+            .INITIALISATION => "Ensure the OS is initialised correctly: Runtime test",
+            .PANIC => "Ensure panic behaviour correctly: Runtime test",
```

Comment:
I think having "Run" here again is unnecessary since it's already in the option's help text and the colon part could be removed if you reword them to "Initialisation runtime tests" and "Panic runtime tests" or something like that